### ISSUE TITLE
check all analysis' if NAs are in dataframe

### ISSUE THREAD

**Issue by tuanle618** [ptl93/AEDA#45]
I noticed that cluster analysis does not work with NAs. I will randomly insert NAs in testthat/base_finishReport file and check, if all analysis' can handle NAs or not. 

In general: How should we handle missing Values? Omit them or do some kind of NA Imputation? Like columns average... ?

**Comment by tuanle618** [comment]
I checked that only for `getClusterAnalysis()` NAs have to be removed.
https://github.com/ptl93/AEDA/blob/528e7de643e5d7466e0f67f4afd8df856bfe6dc8/R/getClusterAnalysis.R#L56-L59

In all `task `I added a parameter for showing message if a dataset contains missing values.
As of now I believe only `getNumSum()` and `getClusterAnalysis()` can only work if we remove the NAs.
In all other tasks there might be default options.
